### ISSUE TITLE
eval_all_combinations setting?

### ISSUE THREAD

**Issue by cicirao** [iit-cs579/main#250]
Could someone explain the sentence, "For each setting, construct a LogisticRegression classifier and compute its cross-validation accuracy for that setting."?
What is the "setting"? Is it the combination of functions? Also, what does it mean to construct a LogisticRegression classifier and compute its cross-validation accuracy for that setting? Is it using the LogisticRegression function?

**Comment by vjonnala** [comment]
A setting is a combination of provided parameters (punct_vals, feature_fns, min_freqs). You can compute accuracy from cross_validation_accuracy() function. Check the arguments for this function to calculate the accuracy.

**Comment by qsong4** [comment]
So what should we use about parameter ‘k’ in cross_validation_accuracy() function ?

**Comment by vjonnala** [comment]
Parameter 'k'  in cross_validation_accuracy() function indicates the number of folds. You can take k to be 5.
Refer 3.1.3.2 in http://scikit-learn.org/stable/modules/cross_validation.html

**Comment by qsong4** [comment]
@vjonnala Thank u! I miss the sentence 'using 5 folds'

**Comment by vjonnala** [comment]
Welcome .. Closing the issue ..

**Comment by cicirao** [comment]
Thank you! But my second question was, what does it mean to construct a LogisticRegression classifier for that setting? Is it using the LogisticRegression function? What would be the parameters? Would those functions be a part of the parameters?
